pkg/workflows/sdk/runtime.go

```diff
@@ -7,9 +7,26 @@ import (
 
 var BreakErr = capabilities.ErrStopExecution
 
+type Emitter interface {
+	// Emit sends a message with the given message and labels to the configured collector.
+	//
+	// TODO(mstreet3): Emit and custmsg.Labeler should be context aware.  Update signature once
+	// WASM can support context.
+	Emit(msg string, labels map[string]any) error
```

Comment:
Ergonomic nit: if you allow a struct to be passed in here users can omit labels if they don't need it

Reply:
Hmm, the responsibilities of the initial `Labeler` struct and this `Emitter` interface seem to have overlap. With the addition of this interface it seems clearer to me that `SendLogAsCustomMessage` should be updated to be called `Emit`. Usages in core will have to be updated as well. 

@cedric-cordenier In core this is one reason why I chose not to have an emitting API with labels as a map. The API can always grow to include labels, but that there are risks there because the Labeler interface can be bypassed if Emit is implemented to only send the labels passed as attributes vs what is stored in the Labeler impl. 

Can we stick to `Emit(msg string)`?